sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdConstants.java

```diff
@@ -859,7 +860,8 @@ public ResourceType type() {
         ItemLocalLSN((short) 0x003C, RntbdTokenType.LongLong, false),
         HasTentativeWrites((short) 0x003D, RntbdTokenType.Byte, false),
         SessionToken((short) 0x003E, RntbdTokenType.String, false),
-        BackendRequestDurationMilliseconds((short) 0X0051, RntbdTokenType.Double, false);
+        BackendRequestDurationMilliseconds((short) 0X0051, RntbdTokenType.Double, false),
+        CorrelatedActivityId((short) 0X0052, RntbdTokenType.Guid, false);
```

Comment:
Do we need to set it at 2 places, here and above?
@xinlian12  to confirm?

Reply:
Yes - one is for request header one for backend header. Same in msdata repository